### PR TITLE
Update version to 1.1.  Minor changes to allow compilation on macOS. …

### PR DESCRIPTION
**Pull request by kwadsten**
… Add labels to PCB.  New /bin output folder.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,6 +6,12 @@ DIRS = src
 
 all:
 	for i in $(DIRS); do $(MAKE) -C $$i || exit 1; done
-
+	cp src/esp/build/partitions_two_ota.bin bin
```
Comment (apuder) [inline Makefile]: Add "mkdir -p bin" before first "cp"

File: Makefile [hunk 2]
```diff
@@ -6,6 +6,12 @@ DIRS = src
...
+	cp src/esp/build/retro-store.bin bin
+	cp src/esp/build/bootloader/bootloader.bin bin
+	
 clean:
 	for i in $(DIRS); do $(MAKE) -C $$i clean || exit 1; done
+	rm bin/partitions_two_ota.bin
```
Comment (apuder) [inline Makefile]: Please do a "rm -rf bin" instead of the three individual "rm"

File: src/ihex2cmd/ihex2cmd.cpp [hunk 3]
```diff
@@ -4,9 +4,9 @@
 
 using namespace std;
 
-#define CHECK(condition, errno) \
+#define TRS_CHECK(condition, errno) \
```
Comment (apuder) [inline src/ihex2cmd/ihex2cmd.cpp]: Could you please change the prefix from "TRS_" to "CMD_"?

File: src/ihex2cmd/ihex2cmd.cpp [hunk 4]
```diff
@@ -4,9 +4,9 @@
...
   if (!(condition)) { \
-    error(errno); \
+    trs_error(errno); \
```
Comment (apuder) [inline src/ihex2cmd/ihex2cmd.cpp]: Same here: "trs_" -> "cmd_"

**Comment by apuder** [comment]
Another suggestion: I like the idea of also committing pre-compiled binaries in /bin. How about renaming this directory to "dist/" and also copy the zipped Gerber files as well as RSCLIENT.CMD to "dist"?

**Comment by kwadsten** [comment]
All sounds good. Will update tomorrow or tues.

File: Makefile (hunk 1 above)
Comment (kwadsten) [reply Makefile]: done

File: Makefile (hunk 2 above)
Comment (kwadsten) [reply Makefile]: done

File: src/ihex2cmd/ihex2cmd.cpp (hunk 3 above)
Comment (kwadsten) [reply src/ihex2cmd/ihex2cmd.cpp]: done

File: src/ihex2cmd/ihex2cmd.cpp (hunk 4 above)
Comment (kwadsten) [reply src/ihex2cmd/ihex2cmd.cpp]: done

**Comment by kwadsten** [comment]
Updates added (I think).  Let me know if it's not correct.

**Comment by kwadsten** [comment]
Trying...  Harder than it should be.  Need to read up a bit more.

**Pull request closed without merging**